dataframe/dataframe.go

```diff
@@ -582,7 +582,7 @@ func (df DataFrame) Capply(f func(series.Series) series.Series) DataFrame {
 }
 
 // Rapply applies the given function to the rows of a DataFrame. Prior to applying
-// the function the elements of each row are casted to a Series of a specific
+// the function the elements of each row are cast to a Series of a specific
```

Comment:
I'm torn on this one, I believe that `casted` is correct in this situation. Trying to find some consensus I arrived here: https://english.stackexchange.com/questions/94565/can-casted-be-the-past-tense-of-cast

Reply:
I think purely grammatically I hate to see casted. It is equivalent to someone saying: "Casted a spell" as opposed to "Cast a spell", but I am sure I can live with either.